eng/ci/host.coldstart.yml

```diff
@@ -20,7 +20,7 @@ parameters:
 - name: perfViewCollectArguments
   displayName: PerfView profile collection arguments (Valid only if "Collect PerfView Profile" is true))
   type: string
-  default: '/NoGui /AcceptEula /NoV2Rundown /NoNGenRundown /NoClrRundown /BufferSizeMB=64 /CircularMB=64 /ThreadTime /Providers=a7044dd6-c8ef-4980-858c-942d972b6250'
```

Comment:
@safihamid suggested removing this from the default value. User can still provide this by overriding the default value when queuing a build, if needed.